Sources/Commands/SwiftTestTool.swift

```diff
@@ -767,11 +769,18 @@ final class ParallelTestRunner {
                         observabilityScope: self.observabilityScope
                     )
                     var output = ""
+                    let start = Date()
                     let success = testRunner.test(outputHandler: { output += $0 })
+                    let end = Date()
```

Comment:
Is it possible to use the new [clock APIs](https://github.com/apple/swift-evolution/blob/main/proposals/0329-clock-instant-duration.md) instead of pulling from Foundation?

Reply:
That looks very promising, but as far as I can tell, it's been accepted but not yet implemented. Since it doesn't affect API of packages, would it make sense to take this PR using the existing facilities but then switch to `SuspendingClock` when it's available?

Reply:
I suggest we take this PR as coded and then switch to a more accurate timekeeping when the API is available in all versions where we want to compile SwiftPM.

Reply:
It has be [implemented in the trunk](https://github.com/apple/swift/blob/541f7725e1b0f4ac2cfc1f1b71a90c5a5c813c1d/stdlib/public/Concurrency/SuspendingClock.swift).

Your suggestion makes sense.